src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+function AddSpanToUl(root) {
```

Comment:
You don't need to create a function for it.  Just get all `li` and the `tree` elements. And use for/of loop